apps/asap-server/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  rules: {
+    'no-use-before-define': 'off',
+    'no-unused-vars': 'off',
```

Comment:
Do you need to disable the rule, before overwriting it?

Reply:
yeah i think so